src/staging/src/pages/sitemap.js

```diff
@@ -25,7 +25,7 @@ const SitemapPage = ({ data }) => {
   return (
     <>
       <Helmet>
-        <title>Site Map | BC Parks</title>
+        <title>BC Parks | Site Map </title>
```

Comment:
The same as above, space before </title>
Also, is this OK to have "BC parks" first?